engine/src/main/java/org/destinationsol/assets/Assets.java

```diff
@@ -208,14 +208,18 @@ public static DSTexture getDSTexture(String path) {
         return getAtlasRegion(path, Texture.TextureFilter.Nearest);
     }
 
-    public static Animation<TextureAtlas.AtlasRegion> getAnimation(TextureAtlas.AtlasRegion original, String path) {
-        if (assetHelper.list(Json.class, path).size() == 0) {
+    public static Animation<TextureAtlas.AtlasRegion> getAnimation(String texturePath, String animationPath) {
```

Comment:
Once again, why do we have two paths, instead of a single texturePath? We can then generate the animationPath from the texturePath (for instance, "core:something" becomes "core:somethingAnimation")

Reply:
I assumed that they might be separate, although that was not needed I now realise, as the convention is already implemented in the code anyway (in the ```SpriteManager``` class). I will just reduce the arguments to one path. This should be in the next commit.